src/rebar_utils.erl

```diff
@@ -407,6 +408,10 @@ abort_if_blacklisted(BlacklistedRegex, OtpRelease) ->
                    [OtpRelease, BlacklistedRegex])
     end.
 
+user_agent() ->
+    {ok, Vsn} = application:get_key(rebar, vsn),
+    ?FMT("Rebar/~s (OTP/~s)", [Vsn, erlang:system_info(otp_release)]).
```

Comment:
Can `otp_release/0` from this module be used instead to get a more detailed OTP version?